packages/wallet/src/redux/protocols/indirect-funding/actions.ts

```diff
@@ -15,3 +16,13 @@ export type FundingRequested = ReturnType<typeof fundingRequested>;
 export { playerA, playerB };
 export type ProcessAction = playerA.Action | playerB.Action;
 export type Action = FundingRequested | ProcessAction | CommonAction;
+
+export function isIndirectFundingAction(action: WalletAction): action is Action {
+  return isCommonAction(action)
+    ? true
+    : isDirectFundingAction(action)
+    ? true
+    : action.type.indexOf('WALLET.INDIRECT_FUNDING') === 0
+    ? true
+    : false;
```

Comment:
This is a kind-of mind-boggling way to write this. Maybe something like the following would be clearer:
```ts
export function isIndirectFundingAction(action: WalletAction): action is Action {
    return isCommonAction(action) || isDirectFundingAction(action) || action.type.indexOf('WALLET.INDIRECT_FUNDING') === 0;
}
```